packages/@aws-cdk/aws-location-alpha/lib/util.ts

```diff
@@ -0,0 +1,9 @@
+import { Names } from 'aws-cdk-lib/core';
+
+export function generateUniqueId(context: any): string {
```

Comment:
Nice 👍 Can you please re-use it in [`place-index.ts`](https://github.com/aws/aws-cdk/blob/f9fd00cd3401e5264518b3409b1130eea976e2db/packages/%40aws-cdk/aws-location-alpha/lib/place-index.ts#L183) and remove the `private` method from that file?

Reply:
Thanks.
I've fixed it.